app/src/main/kotlin/com/saltedge/authenticator/features/authorizations/list/AuthorizationsListViewModel.kt

```diff
@@ -52,7 +52,10 @@ import com.saltedge.authenticator.sdk.tools.crypt.CryptoToolsAbs
 import com.saltedge.authenticator.sdk.tools.keystore.KeyStoreManagerAbs
 import com.saltedge.authenticator.tools.ResId
 import com.saltedge.authenticator.tools.postUnitEvent
-import kotlinx.coroutines.*
+import kotlinx.coroutines.CoroutineDispatcher
```

Comment:
to much new imports in a file that didn't have any changes.

Reply:
Yours favorite `Optimize Imports` did it